packages/avif/codec/Makefile

```diff
@@ -41,10 +41,12 @@ $(OUT_ENC_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMakeLis
 		$(HELPER_MAKEFLAGS) \
 		OUT_JS=$@ \
 		OUT_CPP=$< \
+		STACK_SIZE=5242880 \
+		INITIAL_MEMORY_SIZE=16777216 \
```

Comment:
Setting the default values as seen when previously compiling with Emscripten 2 versions.

In newer Emscripten 3 versions the stack size was decreased to 64KB which causes issues with the AVIF encoding logic.